src/client/mockttp-client.ts

```diff
@@ -153,6 +153,17 @@ export default class MockttpClient extends AbstractMockttp implements Mockttp {
         }
     }
 
+    async pendingMocks(): Promise<any> {
+        try {
+            var requests = await this.requestFromMockServer<any>('/pendingMocks', {
+                method: 'GET'
+            });
+            return requests;
+        } catch (error) {
+            return Promise.reject(error);
+        }
```

Comment:
You don't need the try/catch at all - in an async function, if you just throw an error, it'll return a rejected promise with the error automatically. In fact though, you can go further: you've already got a function that rejects/resolves based on the response - that's exactly what `requestFromMockServer` already does, and you don't even need to `await` it (though you would if you wanted to do any processing on the response).

That means this function can just become:

```ts
pendingMocks(): Promise<any> {
  return this.requestFromMockServer<any>('/pendingMocks', {
    method: 'GET'
  });
}
```

Reply:
You're right, try-catch is unnecessary. I tried this solution and it worked.